Files/UserControls/MultitaskingControl/VerticalTabViewControl.xaml.cs

```diff
@@ -234,20 +239,39 @@ private void TabViewItem_Drop(object sender, DragEventArgs e)
             {
                 e.AcceptedOperation = DataPackageOperation.None;
             }
+            VerticalTabView.CanReorderTabs = true;
+            tabHoverTimer.Stop();
         }
 
-        private void TabViewItem_DragOver(object sender, DragEventArgs e)
+        private void TabViewItem_DragEnter(object sender, DragEventArgs e)
         {
             if (e.DataView.AvailableFormats.Contains(StandardDataFormats.StorageItems))
             {
+                VerticalTabView.CanReorderTabs = false;
                 e.AcceptedOperation = DataPackageOperation.Move;
+                tabHoverTimer.Start();
+                hoveredTabViewItem = sender as TabViewItem;
             }
             else
             {
                 e.AcceptedOperation = DataPackageOperation.None;
             }
         }
 
+        private void TabViewItem_DragLeave(object sender, DragEventArgs e)
```

Comment:
Should this method be used?

Reply:
Are you suggesting that we can use only `TabViewItem_DragEnter`?

Reply:
But won't that cause problems if an item is dragged over, then away from the TabViewItem? This would disable the reordering of the tabs even though it would be a valid operation.

Reply:
No, i am saying, that in `HorizontalMultitaskingControl.xaml` `TabViewItem_DragLeave` is used, but in `VerticalTabViewControl.xaml` this method is missing.

Reply:
Oh I see, sorry I missed that out. Thanks for pointing it out!